demos/common/export_models/export_model.py

```diff
@@ -296,7 +304,13 @@ def export_text_generation_model(model_repository_path, source_model, model_name
         draft_model_dir_name = draft_source_model.replace("/", "-") # flatten the name so we don't create nested directory structure
         draft_llm_model_path = os.path.join(model_repository_path, model_name, draft_model_dir_name)
         if os.path.isfile(os.path.join(draft_llm_model_path, 'openvino_model.xml')):
-                print("OV model is source folder. Skipping conversion.")
+            print("OV model is source folder. Skipping conversion.")
+        elif source_model.startswith("OpenVINO/"):
+            if precision:
+                print("Precision change is not supported for OpenVINO models. Parameter --weight-format {} will be ignored.".format(precision))
+            hugging_face_cmd = "huggingface-cli download {} --local-dir {} ".format(source_model, os.path.join(model_repository_path, model_name))
+            if os.system(hugging_face_cmd):
+                raise ValueError("Failed to download llm model", source_model)    
```

Comment:
is it possible to get error message from the command call? might be useful

Reply:
@mzegla it will be presented anyway